py3status/modules/wifi.py

```diff
@@ -11,9 +11,8 @@
     device: Wireless device name (default "wlan0")
     down_color: Output color when disconnected, possible values:
         "good", "degraded", "bad" (default "bad")
-    format_down: Output when disconnected (default "W: down")
-    format_up: See placeholders below
-        (default "W: {bitrate} {signal_percent} {ssid}")
+    format: Display format for this module
+        (default 'W: [{bitrate} {signal_percent} {ssid}|\?show down]')
```

Comment:
I didn't notice the `?show` bit, is all. IIRC it does not ignore existing `format_up` or `format_down` option with new `format` in place.